### ISSUE TITLE
Improvements in task status and percentage complete change, and user status change

### ISSUE THREAD

**Issue by vinayak-trivedi** [Real-Dev-Squad/website-my#302]
- [ ] In-progress option should get disabled once the user has marked their task 100% complete 
- [ ] once the user gets assigned a task with task leveling their status should be changed to active
- [ ] status should change to idle when if they are not assigned a task
- [ ] If the user has a task assigned they should not be able to change their status to idle, one idea is to not allow them to change status to active or idle by themselves, they can only change it to OOO from idle or active, once they remove the OOO status they will get the status which was there before they changed it to OOO

**Comment by rohan09-raj** [comment]
For the 1st issue, what if user makes the task 100% complete by mistake, is there any going back ??

**Comment by vinayak-trivedi** [comment]
we are already warning the user that this will be marked as 100% completed, we can one more warning that this is an irreplaceable action

**Comment by rohan09-raj** [comment]
> we are already warning the user that this will be marked as 100% completed, we can one more warning that this is an irreplaceable action

Yeah, on the message popup, mention this too that it can't be changed
